pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -454,6 +471,32 @@ func (configgen *ConfigGeneratorImpl) buildInboundClusters(env *model.Environmen
 	return clusters
 }
 
+func (configgen *ConfigGeneratorImpl) findInstance(instances []*model.ServiceInstance,
```

Comment:
this is not correct. What happens to unix domain socket listeners? for UDS, ports have number 0, and the bind specifies the UDS path.

Reply:
Interesting, I didn't test with UDS but it seems fortio supports it. Would be an interesting test.

Reply:
Well - the code is correct for TCP. I'm not sure how UDS fits in - the check is for inbound port which can't be UDS AFAIK. Suggestions ?